INSTALL.txt

```diff
@@ -2,78 +2,107 @@
 Install
 *******
 
-Quick Install
-=============
+PyGraphviz requires:
 
-Get PyGraphviz from the Python Package Index at
-http://pypi.python.org/pypi/pygraphviz
+- Python (version 3.7, 3.8, or 3.9) and
+- `GraphViz <https://www.graphviz.org/>`_ (version 2.42 or later). 
 
-or install it with::
+Below we assume you have the default Python environment already configured on
+your computer and you intend to install ``pygraphviz`` inside of it.  If you want
+to create and work with Python virtual environments, please follow instructions
+on `venv <https://docs.python.org/3/library/venv.html>`_ and `virtual
+environments <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
 
-   pip install pygraphviz
+.. note::
+   If you do not already have a Python environment configured on your computer,
+   please see the instructions for installing the full `scientific Python stack
+   <https://scipy.org/install.html>`_.
 
-and an attempt will be made to find and install an appropriate version
-that matches your operating system and Python version.
 
-You can install the development version (at github.com) with::
+Recommended
+===========
 
-  pip install git://github.com/pygraphviz/pygraphviz.git#egg=pygraphviz
+Linux
+-----
 
+Ubuntu and Debian
+~~~~~~~~~~~~~~~~~
 
-Providing path to graphviz
---------------------------
+.. code-block:: console
 
-We tried our best to discover graphviz location automatically, but if you 
-would like specify specific location for graphviz you may provide additrional parameters to specify graphviz location
+    $ sudo apt-get update
+    $ sudo apt-get install graphviz graphviz-dev
+    $ python3 -m venv ~/venv
+    $ source ~/venv/bin/activate
+    $ pip install --upgrade pip setuptools
+    $ pip install --upgrade pygraphviz 
```

Comment:
I decided to leave the ``sudo apt-get update`` step in for now, since I think the procedure fails on our CI system otherwise (maybe I am misremembering why we had to do that).  At any rate, we need to make sure apt-get users don't install version 2.40 since that doesn't work for us.  So before we remove that line, we should verify that it at least works on our CI system.